.github/CODEOWNERS

```diff
@@ -942,6 +942,9 @@
 # ServiceLabel: %Device Registry %Mgmt
 # ServiceOwners:                                                   @davidemontanari @atastrophic @marcodalessandro @rohankhandelwal @riteshrao
 
+# ServiceLabel: %DevOps - Infrastructure
+# ServiceOwners:                                                   @oolaoluwa
```

Comment:
```suggestion
# PRLabel: %DevOps - Infrastructure
/sdk/devopsinfrastructure/Azure.ResourceManager.*/                 @oolaoluwa

# ServiceLabel: %DevOps - Infrastructure
# ServiceOwners:                                                   @oolaoluwa
```

Reply:
You want to have one entry for pull requests and another for issue triage.   This is the form that you need.